index.js

```diff
@@ -1,8 +1,32 @@
 /** @format */
-
 const harmburger = document.querySelector('.harmburger');
 const navMenu = document.querySelector('.nav-menu');
 const navLink = document.querySelectorAll('.nav-link');
+const modal = document.querySelector('.modal');
+const modalTrigger = document.querySelectorAll('.modal-trigger');
+const wrapper = document.getElementById('wrapper');
+const noScroll = document.querySelector('body');
+
+const projects = [
+  {
+    id: 'project1',
+    title: [
+      'Multi-Post Stories',
+      'Keeping track of hundreds of components website',
+    ],
+    description: [
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    ],
+    link: [
+      'https://duanedave.github.io/Portfolio-Website/',
+      'https://github.com/DuaneDave/Portfolio-Website',
+    ],
+    technologies: ['html', 'bootstrap', 'ruby on rails'],
+    imgSrc: ['asset/popup.png', 'my project'],
+  },
+];
```

Comment:
- [ ] Good job so far :+1: I noticed that just have only one object . Please update your array because from requirement each project should be an object :smile: 
```javascript
const projects = [{
    name: "project One",
    description: "Build portfolio popup window",
    image: "link_to_image",
    technologies: ["html", "JavaScript", "Css"],
    link: "link_to_live_version.com",
},
{
    name: "project Two",
    description: "Build portfolio popup window",
    image: "link_to_image",
    technologies: ["html", "JavaScript", "Css"],
    link: "link_to_live_version.com",
},
{
    name: "project Three",
    description: "Build portfolio popup window",
    image: "link_to_image",
    technologies: ["html", "JavaScript", "Css"],
    link: "link_to_live_version.com",
}
-------------
]

```

Reply:
Pls can you explain to me, there is only one detail for the modal in the Figma and I updated it to show differently on the desktop, how do I get across this suggestion

Reply:
Your portfolio is a way to showcase different projects you built or you will build in the future. They are different from each other with different titles, descriptions, and technologies to be listed on your site. When you click the **See Project** button, you have a pop-up window which is displaying the details for each project.  Each project is an object. If you have five projects displayed on your website, that means you need to have 5 objects stored in your array. In your case you have created the array that is good :+1: but you have only one object and you are using the same object for all your project popup details.

Reply:
so once I add different titles for different projects updated from an object, that fixes it right?

Reply:
The information you get from your modal is the same as you have from your website the only thing is your modal popup gives you more details. You are getting them from the array you create to store the project(object) information.